src/databases/Xmdv/avtXmdvWriter.C

```diff
@@ -35,12 +35,20 @@ using     std::vector;
 //
 //  Programmer: childs -- generated by xml2avt
 //  Creation:   Tue Jul 19 17:12:44 PST 2005
+// 
+//  Modifications:
+//    Justin Privitera, Mon 07 Mar 2022 06:04:53 PM PST
+//    Added gui option to set output precision.
 //
 // ****************************************************************************
 
 avtXmdvWriter::avtXmdvWriter(const DBOptionsAttributes *atts)
 {
     writeOutCoordinates = atts->GetBool("Export coordinates?");
+    outputPrecision = atts->GetInt("Output precision");
+    // Get it in the range 1-20
+    if (outputPrecision > 20) outputPrecision = 20;
```

Comment:
Split the if statement into 4 lines.
I looked at the IEEE standard and it says that double precision is between 15 and 17 digits of accuracy. Let's make the maximum 18 digits.

Reply:
should the default still be 16?

Reply:
Yes, 16 sounds like a good default.